app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -61,6 +61,7 @@
 
             <TextView
               android:id="@+id/story_progress_percentage"
+              style="@style/TextViewCenter"
               android:layout_width="wrap_content"
               android:layout_height="wrap_content"
               android:layout_gravity="center"
```

Comment:
Ditto. This should be removed, right?

Reply:
See comment https://github.com/oppia/oppia-android/pull/4009#discussion_r754973950.